config/schemas/config.yaml

```
@@ -1923,7 +1923,17 @@ properties:
         additionalProperties: false
         properties:
           disableRedirect:
+            title: Harbor registry disable ObjectStorage redirect
```

Nit
```suggestion
            title: Harbor registry disable object storage redirect
```
Same for the description.